### PR TITLE
Laravel 8

### PR DESCRIPTION
**Pull request by mvdnbrk**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -14,7 +14,10 @@ jobs:
       fail-fast: true
       matrix:
         php: [7.2, 7.3, 7.4]
-        laravel: [^7.0]
+        laravel: [^7.0, ^8.0]
+        exclude:
+          -   php: 7.2
+              laravel: 8.0
```
Comment (driesvints) [inline .github/workflows/tests.yml]: Think you have some spaces too much here

Comment (mvdnbrk) [reply .github/workflows/tests.yml]: Hm this does work on another repo...

Comment (driesvints) [reply .github/workflows/tests.yml]: Caret sign was missing it seems.

Comment (mvdnbrk) [reply .github/workflows/tests.yml]: Yeah right ;-) Thanks!

**Comment by driesvints** [comment]
We'll have to adjust the test here to just assert that the Blade Icon components are registered. Doesn't really matters what else is in the array: https://github.com/blade-ui-kit/blade-icons/pull/87/checks?check_run_id=1071544022#step:6:23

**Comment by mvdnbrk** [comment]
`Blade::getClassComponentAliases()`

includes

```
'dynamic-component' => 'Illuminate\View\DynamicComponent'
```

**Comment by mvdnbrk** [comment]
> We'll have to adjust the test here to just assert that the Blade Icon components are registered. Doesn't really matters what else is in the array: https://github.com/blade-ui-kit/blade-icons/pull/87/checks?check_run_id=1071544022#step:6:23

Will take a look at this.

**Comment by mvdnbrk** [comment]
@driesvints Tests are passing now.

**Comment by driesvints** [comment]
Rewrote that a bit. Thanks!

**Comment by driesvints** [comment]
Released https://github.com/blade-ui-kit/blade-icons/releases/tag/0.4.5 with this.

**Pull request merged**